src/components/shared/page-title/page-title.module.css

```diff
@@ -3,14 +3,14 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-.titleRow {
-  overflow: auto;
-}
+/* .titleRow {
+  display: flex;
+} */
```

Comment:
If you're commenting everything out to intentionally get rid of it we could just delete it (it's always in git history)